rust/protocol/src/identity_key.rs

```diff
@@ -1,36 +1,53 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2021 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Wrappers over identity primitives from [crate::curve].
```

Comment:
I think part of the point is that the keys in `crate::curve` are sometimes used for non-identity purposes; these wrappers distinguish the ones that are used as identities.

Reply:
Made this more clear! Let me know if the result sounds good!